zebra-rpc/src/methods.rs

```diff
@@ -69,38 +73,82 @@ pub trait Rpc {
         &self,
         raw_transaction_hex: String,
     ) -> BoxFuture<Result<SentTransactionHash>>;
+
+    /// getblock
+    ///
+    /// Returns requested block by height, encoded as hex.
+    ///
+    /// zcashd reference: <https://zcash.github.io/rpc/getblock.html>
+    ///
+    /// Result:
+    /// {
+    ///      "data": String, // The block encoded as hex
+    /// }
+    ///
+    /// Note 1: We only expose the `data` field as lightwalletd uses the non-verbose
+    /// mode for all getblock calls: <https://github.com/zcash/lightwalletd/blob/v0.4.9/common/common.go#L232>
+    ///
+    /// Note 2: `lightwalletd` only requests blocks by height, so we don't support
+    /// getting blocks by hash.
+    ///
+    /// Note 3: The `verbosity` parameter is ignored but required in the call.
```

Comment:
I'm not sure when you wanted to start writing comments in the standard format from #3638.

Reply:
Not in this PR.

Reply:
https://github.com/ZcashFoundation/zebra/issues/3764